src/client/components/editor/buttons.js

```diff
@@ -7,22 +9,43 @@ const Linkout = require('../document-linkout');
 module.exports = function({ controller, document, bus }){
   let grs = [];
 
+
+  let baseTooltipProps = {
+    show: showNow => {
+      bus.on('showtips', showNow);
+    },
+    hide: hideNow => {
+      bus.on('hidetips', hideNow);
+    },
+    tippy: {
+      zIndex: tippyTopZIndex
+    }
+  };
+
   if( document.editable() ){
     grs.push([
-      h(Tooltip, { description: 'Add entity', shortcut: 'e' }, [
-        h('button.editor-button.plain-button', { onClick: () => controller.addElement().then( el => bus.emit('opentip', el) ) }, [
+      h(Tooltip, { description: 'Help' }, [
```

Comment:
This only shows the help button if the doc is editable.  Thoughts on this?  In some ways it makes sense because it it emits some tooltips related to editing the nodes / edges, but maybe view mode specific tooltips for the nodes / edges would be useful?

Reply:
Makes sense